src/BaseClass.js

```diff
@@ -81,6 +82,16 @@ export default class BaseClass {
     }
   }
 
+  /**
+      @memberof BaseClass
+      @desc If *value* is specified, sets the locale to the specified string and returns the current class instance.
+      @param {String} [*value* = "en-US"]
```

Comment:
similar the implementation in d3plus-axis, can you add description text here that talks about also being able to pass a localization `Object`?